stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/common/InputField.kt

```diff
@@ -38,14 +44,32 @@ public fun InputField(
     innerPadding: Dp = 8.dp,
     decorationBox: @Composable (innerTextField: @Composable () -> Unit) -> Unit,
 ) {
+    var textFieldValueState by remember { mutableStateOf(TextFieldValue(text = value)) }
+
+    val selection = if (textFieldValueState.isCursorAtTheEnd()) {
+        TextRange(value.length)
```

Comment:
A hack to push the cursor to the end after completing the mention from a mention list popup.

Reply:
Added a small comment